src-lib/darknet_image.cpp

```diff
@@ -786,7 +786,46 @@ Darknet::Image Darknet::rgb_mat_to_rgb_image(const cv::Mat & mat)
 	return img;
 }
 
+#if 1
```

Comment:
I would like to see timing results that show this is faster.  If so, let's have a new PR.